client/src/App/utils.tsx

```diff
@@ -95,3 +95,7 @@ export const completedPlansListStatusProp = {
 export const trashPlansListStatusProp = {
   allowedPlanStatus: PlanStatus.RETIRED,
 };
+
+export const planAssignmentProps = {
+  showActivateMission: true,
```

Comment:
should we call them "Mission" in the props? or "Plan"?

Reply:
yeah, I can change that to mission